### DISCUSSION TITLE
Is it possible to hide the day of the month?

### DISCUSSION THREAD

**Discussion by wsw70** [totaldebug/atomic-calendar-revive#877] [Support] [answered]
![image](https://user-images.githubusercontent.com/1345886/208308762-f561ebf3-80ea-426e-a467-bc2b9e8bbb78.png)

I would like to have two `atomic-calendar-revive` cards, one for "today" (`startDaysAhead: 0`) and one for tomorrow  (`startDaysAhead: 1`), one next to each other. This is to replicate my current dashboard (written from scratch) and all my family knows that the one on the left is "today" and the one on the right is "tomorrow". Removing the date of the week would yield a cleaner display (especially since I want to try to style the output)

Is this possible?

**Comment by marksie1988** [comment, marked as answer]
I don't believe it is possible as I always assumed someone would require the date to be shown.

You could possibly try something with card-mod that may allow you to do some clever CSS to hide it?

**Reply by wsw70** [reply]
Thanks for the hint and your **wonderful** card.

I need to work on the CSS anyway as I want the display to be much bigger, bold the hours etc. I had my own Vue-based dash so far but I see that the HA's UI has gone so far ahead that it will be better to adapt cards than to continue building my own.
